backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -659,11 +659,8 @@ bool OpenGLSdlGraphicsManager::notifyEvent(const Common::Event &event) {
 		endGFXTransaction();
 
 #ifdef USE_OSD
-		Common::String message = Common::String::format("%s: %s",
-			_("Stretch mode"),
-			_(stretchModes[index].description)
-			);
-		displayMessageOnOSD(message.c_str());
+		Common::U32String message = _("Stretch mode") + Common::U32String(": ") + _(stretchModes[index].description);
```

Comment:
Why not use `Common::U32String::format`?

Reply:
uhh, yes, you are absolutely correct. This should have been obvious, I don't remember what I must have been thinking back then! Will change it accordingly, a `%S` sounds good.